presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -937,6 +956,13 @@ public void testOptimizeDivideByZero()
     }
 
     @Test
+    public void testTryOptimizeDivideByZero()
+            throws Exception
+    {
+        optimize("TRY (0 / 0)");
```

Comment:
Is there a reason this need to be separate? And it doesn't use `assertOptimizedEquals`?